tests/transport/SubjectInboundTransport.ts

```diff
@@ -19,18 +18,19 @@ export class SubjectInboundTransport implements InboundTransport {
     this.ourSubject = ourSubject
   }
 
-  public async start(agent: Agent) {
+  public async start(agent: AgentContext) {
```

Comment:
Is it not possible to provide the agent here anymore?

Reply:
The problem is that we start transports at DIDComm module's initialize(), which receives the AgentContext as an argument. Before this PR we started transport in `Agent` so we were able to call `transport.start(this)`.

Do you see a potential issue on this change, besides the fact that it breaks the API?